app/App.jsx

```diff
@@ -118,6 +127,7 @@ export default class Application extends React.Component {
               docked={this.state.sidebarDocked}
               messages={messages}
               local={local}
+              language={this.props.local.get('language') ? this.props.local.get('language') : 'en' }
```

Comment:
Thanks, always forget to use these idiomatic JS tricks... Anything else that should be fixed before we merge it?